docs/pages/authentication/what-is-a-signer.mdx

```diff
@@ -31,7 +31,7 @@ User security is our top priority. Alchemy Signer is a non-custodial signer serv
 
 Alchemy Signer is supported out of the box with Smart Wallets via easy to use [React Hooks](/wallets/react/getting-started) and lower-lever [SDKs](/wallets/signer/quickstart). Using Smart Wallets with the Alchemy Signer, you can secure your user's account with an email, social login, or passkeys.
 
-### Read more about signers [here](/wallets/concepts/smart-account-signer).
+### Read more about signers [here](/wallets/signer/what-is-a-signer).
```

Comment:
this one is self-referential, I think Ava was looking to replace this page [with the full one](https://alchemy-preview-5eba56dd-7fcf-4424-8b8d-16d70acb564e.docs.buildwithfern.com/docs/wallets/concepts/smart-account-signer) which used to exist 

if you want to just get this in here I can open a separate PR

Reply:
Ahh good catch, ya sure I'll merge this and you can get it in another PR. Thanks!